tests/test_double_to_rdf.py

```diff
@@ -0,0 +1,69 @@
+"""
+Tests for to_rdf functionality, specifically focusing on double/float handling bugs.
+"""
+
+import json
+import sys
+import os
+import unittest
+
+# Add the lib directory to the path so we can import pyld
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'lib'))
+
+import pyld.jsonld
+
+
+class TestDoubleToRdf(unittest.TestCase):
+    """Test cases for to_rdf functionality with double/float values."""
+
+    def test_offline_pyld_bug_reproduction(self):
+        """Test reproducing the PyLD bug with captured Wikidata data structure."""
+        # This is the exact problematic data structure captured from Wikidata Q399
+        # The bug occurs when PyLD tries to convert this to RDF
+        data = {
+            "@context": {
+                "xsd": "http://www.w3.org/2001/XMLSchema#",
+                "geoLongitude": "http://www.w3.org/2003/01/geo/wgs84_pos#longitude"
+            },
+            "@graph": [
+                {
+                    "@id": "http://www.wikidata.org/entity/Q399",
+                    "geoLongitude": {
+                        "@type": "xsd:double",
+                        "@value": "45"  # This string number causes the PyLD bug
+                    }
+                }
+            ]
+        }
```

Comment:
I think tests should be as minimal as possible.  This could inline xsd prefix, but if this were to be slightly expanded to have various values, xsd: is easier to read.  How about something like:
```suggestion
        data = {
            "@context": {
                "xsd": "http://www.w3.org/2001/XMLSchema#"
            },
            "@graph": [
                {
                    "@id": "ex:1",
                    "ex:p": {
                        "@type": "xsd:double",
                        "@value": "45"
                    }
                }
            ]
        }
```